releasenotes/index.md

```diff
@@ -6,6 +6,7 @@ permalink: /releasenotes/
 
 The Release Notes and Javadocs for every major release of Apache jclouds are listed chronologically below.
 
+* [1.9.2](/releasenotes/1.9.2) | [Javadoc](/reference/javadoc/1.9.x/)
```

Comment:
Looking at other releases, it looks like we don't publish javadocs for every minor?
http://jclouds.apache.org/reference/javadoc/

Should we deploy all missing versions?